mne_bids/tests/test_mne_bids.py

```diff
@@ -254,6 +265,10 @@ def test_vhdr():
     assert data['units'][data['name'].index('FP1')] == 'µV'
     assert data['units'][data['name'].index('CP5')] == 'n/a'
 
+    # check events.tsv is written
+    events_tsv_fname = channels_tsv_name.replace('channels', 'events')
+    assert op.exists(events_tsv_fname)
```

Comment:
What is this assertion supposed to check? At this point, we already know that `channels_tsv_name` is a file and it exists (because we just managed to read data from it some rows above

Reply:
yes, but here we are looking at events_tsv_fname not channels_tsv_fname

Reply:
ok! I misunderstood this ... seems fine. :-)